### PR TITLE
[SOFT-201] Cruise Control

### PR DESCRIPTION
**Pull request by GeraldNDA**
It is what it is

### CODE REVIEW

**Comment by GeraldNDA** [comment]
Pretty sure these tests are incomplete, so lemme know if you notice cases I missed. Also I'll fix formatting and linting tonight/tomorrow

File: projects/centre_console/src/main.c [hunk 1]
```diff
@@ -52,6 +52,7 @@ void prv_init_fsms() {
   power_off_sequence_init(&s_off_sequence_storage);
   power_fsm_init(&s_power_fsm_storage);
   drive_fsm_init(&s_drive_fsm_storage);
+  cruise_rx_init();
```
Comment (g-jessmuir) [inline projects/centre_console/src/main.c:55]: is this necessary?

**Review by g-jessmuir — APPROVED** [review]
only thing I really see is testing increase/decrease at min/max limits for cruise, but it's probably fine, and linting I guess

File: projects/centre_console/src/main.c (hunk 1 above)
Comment (GeraldNDA) [reply projects/centre_console/src/main.c:55]: oof mistake lol

**Pull request merged**